.gitlab/build_oci_package.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+
+npm pack
+
+mkdir -p packaging/sources
+
+npm install --prefix ./packaging/sources/ dd-trace-*.tgz
+
+rm packaging/sources/*.json # package.json and package-lock.json are unneeded
```

Comment:
`package.json` is definitely needed, since it contains information _other_ than dependencies that's used at runtime.

Reply:
Ah, gotcha.